js/mexc3.js

```diff
@@ -0,0 +1,3659 @@
+                        'post': {
+                            'position/change_margin': 2,
+                            'position/change_leverage': 2,
+                            'order/submit': 2,
+                            'order/submit_batch': 40,
+                            'order/cancel': 2,
+                            'order/cancel_with_external': 2,
+                            'order/cancel_all': 2,
+                            'account/change_risk_level': 2,
+                            'planorder/place': 2,
+                            'planorder/cancel': 2,
+                            'planorder/cancel_all': 2,
+                            'stoporder/cancel': 2,
+                            'stoporder/cancel_all': 2,
+                            'stoporder/change_price': 2,
+                            'stoporder/change_plan_price': 2,
+                        },
+                    },
+                },
+                'spot2': {
+                    'public': {
+                        'get': {
+                            'market/symbols': 1,
+                            'market/coin/list': 2,
+                            'common/timestamp': 1,
+                            'common/ping': 1,
+                            'market/ticker': 1,
+                            'market/depth': 1,
+                            'market/deals': 1,
+                            'market/kline': 1,
+                            'market/api_default_symbols': 2,
+                        },
+                    },
+                    'private': {
+                        'get': {
+                            'account/info': 1,
+                            'order/open_orders': 1,
+                            'order/list': 1,
+                            'order/query': 1,
+                            'order/deals': 1,
+                            'order/deal_detail': 1,
+                            'asset/deposit/address/list': 2,
+                            'asset/deposit/list': 2,
+                            'asset/address/list': 2,
+                            'asset/withdraw/list': 2,
+                            'asset/internal/transfer/record': 10,
+                            'account/balance': 10,
+                            'asset/internal/transfer/info': 10,
+                            'market/api_symbols': 2,
+                        },
+                        'post': {
+                            'order/place': 1,
+                            'order/place_batch': 1,
+                            'asset/withdraw': 2,
+                            'asset/internal/transfer': 10,
+                        },
+                        'delete': {
+                            'order/cancel': 1,
+                            'order/cancel_by_symbol': 1,
+                            'asset/withdraw': 2,
+                        },
+                    },
+                },
+            },
+            'timeframes': {
```

Comment:
I guess this is a leftover and should be removed, we have the correct "timeframes" object inside options

Reply:
no, it's not leftover. I'll drop a link for more clarification, this was confusing to me too before I understood the reason behind this.
( here link: https://discord.com/channels/690203284119617602/921046068555313202/957697009324789801 )

so, in case when there are 2+ market types with possible different timeframe values-groups, then in main `options.timeframes` object, we set unified-to-exchange-specific mappings. i suggest read through Igor's comments, and if you'll have a bit doubts in understanding, I'll write-up small paragraph, probalby to be added in manuals.